__tests__/app.test.js

```diff
@@ -425,3 +425,46 @@ describe('GET /api/users', () => {
       })
   })
 })
+describe('GET /api/articles (sorted)', () => {
+  it('GET: 200 - Should sort articles by date in descending order by default', () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then(({ body }) => {
+        expect(body.articles).toBeSortedBy('created_at', {descending: true});
+      })
+  })
+  it('GET: 200 - Should sort articles by selected column in ascending order', () => {
+    return request(app)
+    .get('/api/articles?sort_by=title&order=asc')
+    .expect(200)
+    .then(({ body }) => {
+        expect(body.articles).toBeSortedBy('title', { ascending : true});
```

Comment:
toBeSortedBy doesn't have an ascending option: https://www.npmjs.com/package/jest-sorted
Instead it defaults to ascending.